dev/io.openliberty.io.smallrye.config/src/io/openliberty/microprofile/config/internal/extension/OLSmallRyeConfigProviderResolver.java

```diff
@@ -161,4 +164,28 @@ private static String getApplicationName() {
         return applicationName;
     }
 
+    @Override
+    public String getIntrospectorDescription() {
```

Comment:
Would it help to differentiate this description and name from the AbstractProviderResolver intropector (far)above?

Reply:
I think it makes sense for both introspectors to have the same name since that decides the output filename in the dump. Both introspectors are essentially the same, just for mpConfig 1.x and 2.x and I think it makes more sense for the introspection output to be in the same place for both versions.

Potentially we could put which introspector was used in the description, though we couldn't easily include which specific version of the config feature is in use since the introspector doesn't know that information, so we could only put whether it was 1.x or 2.x.